### PR TITLE
chore: quoting labels in issue templates

### PR DESCRIPTION
**Pull request by fabiomadge**

### CODE REVIEW

**Comment by davidcok** [comment]
I find the title prefixes useful as well. Is there harm in keeping them?

**Comment by fabiomadge** [comment]
I find them to be visually unappealing and disruptive to the flow when glancing at title line by line.

**Comment by davidcok** [comment]
I find them helpful when scanning line by line

**Comment by MikaelMayer** [comment]
I think this is a duplicate PR from https://github.com/dafny-lang/dafny/pull/3108
I agree with Fabio, we can always filter them by label, and I prefer to see a full title.

**Comment by fabiomadge** [comment]
@MikaelMayer Yes, thanks for pointing it out! Let's revisit the other changes, once #3108 is merged.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 name: Bug Report
 description: File a bug report
-labels: [kind: bug]
+labels: "kind: bug"
```
Comment (fabiomadge) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:3]: Looks like we need apostrophes here, as the current version doesn't create any labels.

**Review by MikaelMayer — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 2]
```diff
@@ -25,21 +25,21 @@ body:
...
       description: Describe the issue here. What did you expect instead?
-      placeholder: Tell us what you see!
+      placeholder: Tell us what you wish to see!
     validations:
       required: true
   - type: dropdown
     id: operating-systems
     attributes:
-      label: What type of Operating System are you seeing the problem on?
+      label: What type of Operating System are you experiencing the problem on?
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:42]:
```suggestion
      label: What type of operating system are you experiencing the problem on?
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 3]
```diff
@@ -25,21 +25,21 @@ body:
...
+      label: Command to run and resulting output
       description: "Please paste here what you ran (command, vscode) and what were your results that you think ought to be different:"
       render: code
   - type: textarea
     id: what-happened
     attributes:
       label: What happened?
       description: Describe the issue here. What did you expect instead?
-      placeholder: Tell us what you see!
+      placeholder: Tell us what you wish to see!
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:36]: Why?

Comment (fabiomadge) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yml:36]: I guess it's both, given that the description ask for both observed and expected behaviour. How about:

> Tell us what you see and wish to see!

**Review by davidcok — APPROVED** [review]

**Pull request merged**